src/scheduler/job_info.c

```diff
@@ -5095,29 +5095,28 @@ nspec **create_res_released_array(status *policy, resource_resv *resresv)
  * @note only uses RASSN resources on the sched_config resources line
  * 
  * @param policy - policy info
- * @param res_released - resources_released array to accumulate
+ * @param pjob -  resource reservation structure
  * @return resource_req *
  * @retval newly created resreq_rel array
  * @retval NULL on error
  */
-resource_req *create_resreq_rel_list(status *policy, nspec **res_released)
+resource_req *create_resreq_rel_list(status *policy, resource_resv *pjob)
 {
-	int i;
 	resource_req *resreq_rel = NULL;
 	resource_req *rel;
 	resource_req *req;
-	if(policy == NULL || res_released == NULL)
+	if(policy == NULL)
```

Comment:
you probably wanna check that pjob is not NULL before dereferencing it later.

Reply:
I'll add that check